prow/cmd/entrypoint/README.md

```diff
@@ -16,8 +16,11 @@ as JSON in the `$ENTRYPOINT_OPTIONS` environment variable, which has the form:
         "/bin/ls",
         "-la"
     ],
-    "timeout_minutes": 10,
+    "timeout": 7200000000000,
```

Comment:
Will this parse `"timeout:" "10m"`?

Reply:
No, and that is what I was originally trying that was impossible. It has to be the `int64`. We can deal with that I think because JSON is the machine way to interact and machines don't care.